package.json

```
@@ -8,6 +8,7 @@
   "license": "MIT",
   "private": false,
   "scripts": {
+    "repl": "node repl.js",
```

Probably a good idea to build first?
```suggestion
    "repl": "yarn build && node repl.js",
```